test/check_gemm.hh

```diff
@@ -13,11 +13,21 @@
 
 #include <limits>
 
+namespace std {
+  template <> class numeric_limits<blas::float16> {
+  public:
+    static blas::float16 epsilon() {
+        // Value coming from MAGMA testing/testing_hgemm.cpp
+        return blas::float16( 0.00097656 );
+    }
+  };
+}; // namespace std
```

Comment:
4-space indent
newline after `template <>` so `class` begins on a new line.
Namespaces don't end in `;` just `}  // namespace std`.